buildSrc/src/main/groovy/io.micronaut.build.internal.convention-base.gradle

```diff
@@ -46,6 +46,15 @@ tasks.withType(Jar).configureEach {
     preserveFileTimestamps = false
 }
 
+configurations.all {
+    resolutionStrategy.eachDependency { DependencyResolveDetails details ->
+        if (details.requested.group == 'org.codehaus.groovy') {
```

Comment:
So while this will work to build and execute this module, this won't be propagated to consumers.

Reply:
I think that's ok...  It's only stopping the micronaut-build internal plugin pulling in the codehaus dependency here

Once micronaut-build is updated, I believe we can get rid of this